### ISSUE TITLE
Bring active button state in line with the mocked image

### ISSUE THREAD

**Issue by jpcody** [kacyhulme/timeline#2]

**Comment by kacyhulme** [comment]
Can you elaborate on this. I am not exactly clear. I am thinking it may be in regard to the purple button and the beveled edge.

**Comment by jpcody** [comment]
Yes! Thanks for asking for clarification. Here are the mock and button side-by-side:

![](http://d.pr/i/1ehGx.png)

There are a few things I'd love brought in line:
- [ ] The border
- [ ] The shadowing
- [ ] The type (covered in #1)

**Comment by kacyhulme** [comment]
Ok. got it.
